lc-hax/Scripts/Modules/Possession/Controllers/BrackenController.cs

```diff
@@ -1,21 +1,25 @@
 using Hax;
 
 enum FlowerMan {
-    DEFAULT = 0,
-    STAND = 1
+    SCOUTING = 0,
+    STAND = 1,
+    ANGER = 2,
 }
 
+
 internal class BrackenController : IEnemyController<FlowermanAI> {
     internal void UsePrimarySkill(FlowermanAI enemyInstance) {
-        if (!enemyInstance.carryingPlayerBody) return;
-
-        _ = enemyInstance.Reflect().InvokeInternalMethod("DropPlayerBody");
+        if (!enemyInstance.carryingPlayerBody) {
+            enemyInstance.SetBehaviourState(FlowerMan.ANGER);
+        }
+        else
+            _ = enemyInstance.Reflect().InvokeInternalMethod("DropPlayerBody");
         enemyInstance.DropPlayerBodyServerRpc();
```

Comment:
Why are we dropping the body twice?

Reply:
That’s the client version, to both force the bracken animation to do it

Reply:
Don't we only need one? Since the server RPC will call the client RPC?

Reply:
```cs
    [ServerRpc]
    public void DropPlayerBodyServerRpc()
    {
        NetworkManager networkManager = base.NetworkManager;
        if ((object)networkManager == null || !networkManager.IsListening)
        {
            return;
        }

        if (__rpc_exec_stage != __RpcExecStage.Server && (networkManager.IsClient || networkManager.IsHost))
        {
            if (base.OwnerClientId != networkManager.LocalClientId)
            {
                if (networkManager.LogLevel <= LogLevel.Normal)
                {
                    Debug.LogError("Only the owner can invoke a ServerRpc that requires ownership!");
                }

                return;
            }

            ServerRpcParams serverRpcParams = default(ServerRpcParams);
            FastBufferWriter bufferWriter = __beginSendServerRpc(2817453984u, serverRpcParams, RpcDelivery.Reliable);
            __endSendServerRpc(ref bufferWriter, 2817453984u, serverRpcParams, RpcDelivery.Reliable);
        }

        if (__rpc_exec_stage == __RpcExecStage.Server && (networkManager.IsServer || networkManager.IsHost))
        {
            DropPlayerBodyClientRpc();
        }
    }

    [ClientRpc]
    public void DropPlayerBodyClientRpc()
    {
        NetworkManager networkManager = base.NetworkManager;
        if ((object)networkManager != null && networkManager.IsListening)
        {
            if (__rpc_exec_stage != __RpcExecStage.Client && (networkManager.IsServer || networkManager.IsHost))
            {
                ClientRpcParams clientRpcParams = default(ClientRpcParams);
                FastBufferWriter bufferWriter = __beginSendClientRpc(1942952026u, clientRpcParams, RpcDelivery.Reliable);
                __endSendClientRpc(ref bufferWriter, 1942952026u, clientRpcParams, RpcDelivery.Reliable);
            }

            if (__rpc_exec_stage == __RpcExecStage.Client && (networkManager.IsClient || networkManager.IsHost))
            {
                DropPlayerBody();
            }
        }
    }
```

Reply:
Yeah no need, i dont know why controlcompany does that lol